doc/quickstart/geoserver_quickstart.rst

```diff
@@ -29,35 +29,32 @@ This Quick Start describes how to:
   * add a layer from a NetCDF file
 
 .. contents:: Contents
+   :local:
   
-Start |GS|
-================================================================================
+Start Geoserver
+===============
 
-#. Select the |osgeolive-appmenupath-geoserver| in the menu.
-#. The application will take a few moments to start up and will open a web page at http://localhost:8082/geoserver/web 
+#. From the Start menu, select :menuselection:`Geospatial --> Web Services --> GeoServer --> Start GeoServer`. The application will take a few moments to start up and will open a web page at http://localhost:8082/geoserver/web 
```

Comment:
Note, I couldn't get the service to start on either 13.0 or version 12.0. I reviewed the doc without being able to truly run through the procedure.

Reply:
@fgdrf @flicstar has anyone managed to get geoserver to start on OSGeoLive? We might have great docs now, but they are not very helpful if they are explaining an application which doesn't work.
@fgdrf are you be in a position to download osgeolive 13.0 and see  if there is a problem we need fixing? We might need to raise an issue against the OSGeoLive software, or GeoServer software.

Reply:
GeoServer works fine both on 12.0 and 13.0.
It could be a matter of RAM allocation to the VM.

Reply:
Thanks @kalxas, based on your comment, I will consider geoserver quickstart good for merging.